Sources/Build/BuildOperationBuildSystemDelegateHandler.swift

```diff
@@ -465,6 +470,61 @@ public final class BuildExecutionContext {
     }
 }
 
+final class WriteAuxiliaryFileCommand: CustomLLBuildCommand {
+    override func getSignature(_ command: SPMLLBuild.Command) -> [UInt8] {
+        guard let buildDescription = self.context.buildDescription else {
+            return []
+        }
+        guard let tool = buildDescription.copyCommands[command.name] else {
+            return []
+        }
+
+        let encoder = JSONEncoder.makeWithDefaults()
+        var hash = Data()
+        hash += try! encoder.encode(tool.inputs)
+        hash += try! encoder.encode(tool.outputs)
```

Comment:
can we avoid `try!` here?

Reply:
Yah, I guess, this was just copy/paste from existing code.